Sources/Specs/ViewSpec.swift

```diff
@@ -0,0 +1,19 @@
+import SnowManLogic
+import Swiftest
+
+class ViewSpec: Swiftest.Spec {
+  	let spec = describe("view logic") {
+  		var ui: Receivable!
+  		var view: View!
+  		before() {
+  			ui = MockUI()
+  			view = View(ui: ui)
+  		}
```

Comment:
same thing about all the other setup stuff, don't add all these setup steps until you need them.